solarwinds_apm/sampler.py

```diff
@@ -192,21 +154,40 @@ def calculate_liboboe_decision(
                 status_msg,
                 auth_msg,
                 status,
-            ) = (
-                0,
-                0,
-                0,
-                0,
-                0.0,
-                0.0,
-                0,
-                0,
-                "",
-                "",
-                0,
-            )
+            ) = self.oboe_settings_api.getTracingDecision()
```

Comment:
Do the arguments to this call not matter ? Actually curious if passing it the same thing as the non-lambda version vs passing it nothing makes any difference or if I should be doing it differently on my side of things

Reply:
Ah good question. Tried it now and yes, this method does not explode if passed all the same non-lambda args (though it's a drop decision). Works if no args at all of course.

It does error out if I try other combinations of args:`Wrong number or type of arguments for overloaded function`